.env.example

```diff
@@ -21,8 +24,5 @@ ETH_GAS=15000000
 ## Solidity Compiler Version ##
 SOLC_VERSION=0.8.18
 
-## AJNA token address for target chain ##
-AJNA_TOKEN=0xaadebCF61AA7Da0573b524DE57c67aDa797D46c5
```

Comment:
This wasn't being used anywhere.  Had it commented-out in my env file for many months.